src/main/scala/ai/starlake/extract/JDBCUtils.scala

```diff
@@ -777,85 +891,75 @@ object JDBCUtils extends LazyLogging {
                 connection,
                 deltaRow,
                 Some(partitionColumnType),
-                colStart,
-                colEnd
+                colNameQuoteAudit,
+                colNameQuoteAudit
               )
             }
 
         }
     }
+    forkJoinTaskSupport.foreach(_.forkJoinPool.shutdown())
+  }
+
+  private def getStringPartitionFunc(jdbcUrl: String): Option[String] = {
```

Comment:
OK I get it we partition the table based on a hash of the string modulo the number of partitions